### PR TITLE
COG 30

### PR DESCRIPTION
**Pull request by dashaylan**
Per COG-30, adds a new view displaying metadata from CHORD in a multitable format. This data is NOT filtered by specific variant as specified in COG-5, but just an implementation of the data rendering. The getMetadata function will need to be altered to accommodate the specific data in a later PR.

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -5,6 +5,8 @@
 // API URL where the Dashboard get all the data
 const BASE_URL = 'http://ga4ghdev01.bcgsc.ca:20127';
 
+export const CHORD_METADATA_URL = ' http://ga4ghdev01.bcgsc.ca:4000';
```
Comment (haoyuanli) [inline src/constants/constants.js:8]: why is there a space before `http`

Comment (dashaylan) [reply src/constants/constants.js:8]: I have no idea

File: src/components/Tables/ChordMetadataTable.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+              <thead>
+                {headerGroups.map((headerGroup) => (
+                  <tr {...headerGroup.getHeaderGroupProps()}>
+                    {headerGroup.headers.map((column) => (
+                      <th {...column.getHeaderProps()}>
+                        <div>
+                          {column.canGroupBy ? (
+                          // If the column can be grouped, add a toggle
+                            <span {...column.getGroupByToggleProps()}>
+                              {column.isGrouped ? '🛑 ' : '👊 '}
```
Comment (fcoralsasso) [inline src/components/Tables/ChordMetadataTable.jsx:127]: Nice.

File: src/views/ChordTableApp.jsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+function getMetadataData(setData, setPhenopackets) {
+  return fetch(`${CHORD_METADATA_URL}/api/individuals`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
```
Comment (fcoralsasso) [inline src/views/ChordTableApp.jsx:39]: Nice. I am going to do the same on my side.

**Pull request merged**